ssm/agents.py

```diff
@@ -259,6 +262,8 @@ def run_sender(protocol, brokers, project, token, cp, log):
 
     log.info('SSM has shut down.')
     log.info(LOG_BREAK)
+    if sender_failed is True:
```

Comment:
```suggestion
    if sender_failed:
```
Let's use the compact format as it's clear enough (as we chose a good variable name).